app/services/scene-collections/scene-collections.ts

```diff
@@ -668,16 +670,12 @@ export class SceneCollectionsService extends Service implements ISceneCollection
     if (this.autoSavePromise) await this.autoSavePromise;
   }
 
-  private async setActiveCollection(id: string) {
+  private async setActiveCollection(id: string, informServer = true) {
```

Comment:
What is `informServer doing?  I don't see it being used.

Reply:
Forgot to delete it from some experiments